### DISCUSSION TITLE
Installation on Windows

### DISCUSSION THREAD

**Discussion by Fil1989** [dafny-lang/dafny#2517] [General]
Hi everyone. I am trying to install dafny on Windows 10. I am absolutely following the instructions from the readme file, but finally, when I run the command Dafny.exe, the result is:

Dafny.exe: command not found

Any thoughts?

**Comment by cpitclaudel** [comment]
I would recommend using VSCode directly instead of the command line. 

If you do want to use the command line, you need to make sure that the directory in which you placed Dafny is in your PATH environment variable (so that Windows can find it), or that you call Dafny.exe from the folder that contains Dafny.

Additionally, some versions of Windows will require you to "trust" the downloaded zip after you download it.  Download Dafny, then right click the zip, click "properties", and click "Unblock":

![image](https://user-images.githubusercontent.com/8181630/181791861-7ed24cec-5b3c-4fb4-aaad-71379c0cb148.png)

After that you can unzip and keep going.
